controllers/kustomization_decryptor.go

```diff
@@ -45,13 +45,17 @@ import (
 	intkeyservice "github.com/fluxcd/kustomize-controller/internal/sops/keyservice"
 )
 
-const DecryptionProviderSOPS = "sops"
+const (
+	DecryptionProviderSOPS = "sops"
+	vaultTokenFileName     = "sops.vault-token"
```

Comment:
```suggestion
	DecryptionVaultTokenFileName = "sops.vault-token"
```

Reply:
Why do you want to export it?

Reply:
Because it's static value being checked on and thus arguably part of a contract.